org.kde.merkuro.json

```diff
@@ -0,0 +1,972 @@
+{
+    "id": "org.kde.merkuro",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "6.7",
+    "base": "io.qt.qtwebengine.BaseApp",
+    "base-version": "6.7",
+    "sdk": "org.kde.Sdk",
+    "sdk-extensions": [
+        "org.freedesktop.Sdk.Extension.rust-stable"
+    ],
+    "command": "run_merkuro.sh",
+    "finish-args": [
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--device=dri",
+        "--filesystem=~/.gnupg:create",
+        "--filesystem=xdg-run/gnupg:ro",
```

Comment:
use `--socket=gpg-agent` is better as that extracts the socket path from gpg itself while this expects it to be in xdg-run https://docs.flatpak.org/en/latest/sandbox-permissions.html#standard-permissions